packages/headless/package.json

```diff
@@ -33,19 +33,19 @@
     "@nestjs/swagger": "^5.2.1",
     "@nestjs/testing": "^8.4.7",
     "@nestjsi/class-validator": "^0.1.4",
-    "@types/sequelize": "^4.28.13",
```

Comment:
types/sequelize is used twice. remove one. i'd suggest to copy the package.json from the base branch and hse that in your code. as you didn’t add/remove any dependency, no change should be in package json and yarn.lock file